sections/material-design.liquid

```diff
@@ -0,0 +1,119 @@
+{{ 'component-material-design.css' | asset_url | stylesheet_tag }}
+<script src="{{ 'material-design.js' | asset_url }}" defer="defer"></script>
+
+<div class="page-width">
+  <div class="material-design__container">
+
+    <div class="material-design__container-header" style="background-image: url({{ section.settings.material_design_image | img_url: 'original' }})">
+      <div class="material-design__text">
+        <h3 class="material-design__title">
+          {{ section.settings.material_design_title }}
+        </h3>
+        <p class="material-design__desctiption">
+          {{ section.settings.material_design_description }}
+        </p>
+      </div>
+    </div>
+
+    <div class="material-design__multicolumn">
+      <div class="material-design__multicolumn-item">
+        <div class="multicolum-item__title">Delivery time</div>
+        <div>
+          <span>Product will be delivered on ...</span>
+          <p class="delivery-time__container">
+            <span
+              id="delivery-days"
+              class="delivery-time__days-left"
+              data-delivery-time="{{ section.settings.material_design_delivery_time  }}">
+              {{ section.settings.material_design_delivery_time }}
+            </span>
+            <span id="delivery-suffix" class="delivery-time__days"></span>
+          </p>
+          <span id="delivery-date" class="delivery-time__date"></span>
+        </div>
```

Comment:
I think we should only show this column when metafield is filled out. Same like for rest of the columns below.

Reply:
It was actually my mistake that there was no support for the lack of delivery time, thanks for pointing that out